doc/planning_scene_ros_api/src/planning_scene_ros_api_tutorial.cpp

```diff
@@ -171,7 +171,30 @@ int main(int argc, char** argv)
   // and send the diffs to the planning scene via a service call:
   auto request = std::make_shared<moveit_msgs::srv::ApplyPlanningScene::Request>();
   request->scene = planning_scene;
-  planning_scene_diff_client->async_send_request(request);
+  std::shared_future<std::shared_ptr<moveit_msgs::srv::ApplyPlanningScene_Response>> response_future;
+  response_future = planning_scene_diff_client->async_send_request(request);
+
+  // wait for the service to respond
+  std::chrono::seconds wait_time(3);
+  std::future_status fs = response_future.wait_for(wait_time);
+  if (fs == std::future_status::timeout)
+  {
+    RCLCPP_WARN(LOGGER, "Service timed out.");
```

Comment:
Changed to error messages. I'm not exiting the program here though, since this is technically a duplicate operation of the publish above it so the object can still be added properly.